lib/generators/simple_form/templates/_form.html.erb

```diff
@@ -1,5 +1,6 @@
 <%%= simple_form_for(@<%= singular_table_name %>) do |f| %>
   <%%= f.error_notification %>
+  <%%= f.error_notification message: f.object.errors[:base].to_sentence if f.object.errors[:base].present? %>
```

Comment:
You can use `f.error :base` instead.

Reply:
Does f.error :base automatically check whether an error is actually present?
Don't want to generate extra markup for no error message.

IE, Should the line say 
`<%%= f.error :base %>`
or
`<%%= f.error :base if ... %>`

See here for more info:
https://github.com/plataformatec/simple_form/issues/1330

Reply:
> Does f.error :base automatically check whether an error is actually present?

Yes, it is.

> Don't want to generate extra markup for no error message.

According to my tests, it doesn't generate extra markup. The only difference that I saw was that with `f.error` uses `span` tag with class `error` instead `div` with class `error_notification`.

```ruby
f.error_notification
=> <div class="error_notification">Custom validation</div>

f.error
=> <span class="error">Custom validation</span>
```